src/hooks/useToast.tsx

```diff
@@ -158,18 +158,18 @@ const ToastContent = styled.div`
   align-items: flex-start;
 `;
 const ToastTitle = styled.div`
-  font-size: 1.1rem;
-  font-weight: 600;
+  font-weight: 500;
+  font-size: 18px;
```

Comment:
UI looks good, Since all the size units used rem and em...is it possible to use same unit(rem) here?

Reply:
yes we can use in rem